arch/cpu/msp430/Makefile.msp430

```diff
@@ -182,6 +182,9 @@ endif # SMALL
 
 endif # IAR
 
+# Define the `_stack` symbol used by the stack check library to be equal to `_end`
+LDFLAGS += -Wl,--defsym=_stack=_end
```

Comment:
Not sure, but I somehow think it's possible to achieve this inside the linker script?

Reply:
The msp430 linker scripts are currently not shipped with Contiki. I don't think that adding a modified version of the linker script to the OS is preferable to one-line change in Makefile.

Reply:
Gocha OK, how about the linker scripts for other CPUs though?

Reply:
Already done for cc2538 and cc26xx; can do that for nrf52. For the other platforms (JN516x and msp430) Makefiles.

Reply:
You ok with the change now?

Reply:
All good 👍